DataFormats/HcalRecHit/src/HFQIE10Info.cc

```diff
@@ -1,81 +1 @@
-#include <algorithm>
 #include "DataFormats/HcalRecHit/interface/HFQIE10Info.h"
-
-const unsigned HFQIE10Info::N_RAW_MAX;
-const HFQIE10Info::raw_type HFQIE10Info::INVALID_RAW;
```

Comment:
these were added in #16041 to solve a compilation error in clang.
I see that all use cases in this .cc file were moved out.
Still, the N_RAW_MAX is used elsewhere, e.g. RecoLocalCalo/HcalRecAlgos/src/HFPreRecAlgo.cc, which may look like need an address of this const, e.g. ``` const int nStore = std::min(nRead, static_cast<int>(HFQIE10Info::N_RAW_MAX)); ```).

@Dr15Jones @smuzaffar is it safe to assume that the jenkins build tests are enough or do we need some special compilation test?

Reply:
I was hoping to get a response to this 
> @Dr15Jones @smuzaffar is it safe to assume that the jenkins build tests are enough or do we need some special compilation test?

Reply:
@smuzaffar I believe only you can answer the question.